cityofaustin/janis
# 4634 Update Janis "😃 Getting Started" Section

# Description

This PR, dose not change any code that effects Janis. Rather, it only updates the "Getting started" section in this repositories readme file. 

The additional documentation covers some additional steps to getting started with janis. 

# How can this be tested?

Ideally, start with a new computer, go through the steps and see if you can get a local branch with production data. 

# Checklist:
